src/main.cpp

```diff
@@ -310,11 +322,26 @@ int main(int argc, char * argv[])
     if (!(ErrorDialogHandler::instance()->checkError())) {
         qDebug() << "Displaying mixxx";
         mixxx->show();
+        splash.finish(mixxx);
 
         qDebug() << "Running Mixxx";
         result = a.exec();
     }
 
+    QMessageBox cleaningUpBox;
+    cleaningUpBox.setStandardButtons(0);
+    cleaningUpBox.setWindowTitle(QObject::tr("Exiting Mixxx"));
+    cleaningUpBox.setText(QObject::tr(
+            "Cleaning up the library. This may take some time.\n"
+            "Do not restart Mixxx or shutdown the device until\n "
+            "this box has disappeared."));
```

Comment:
While it is nice that mixxx runs on an eee pc, that is not our target machine and its startup times are not relevant.

Daniel once again we are arguing without end because nothing I say will convince you that a PR is misguided.  You asked for a review, I say "no merge."  I also suggested concrete alternatives that will help solve the problem you encountered.  Either you can respect my opinion or you should just forget the review process and commit everything you write directly to trunk.